src/other/precompile.jl

```diff
@@ -5,5 +5,5 @@ function _precompile_()
     Base.precompile(Tuple{typeof(pretty),DefaultStyle,CSTParser.EXPR,State})
     Base.precompile(Tuple{typeof(pretty),YASStyle,CSTParser.EXPR,State})
     Base.precompile(Tuple{typeof(pretty),BlueStyle,CSTParser.EXPR,State})
-
+    Base.precompile(Tuple{typeof(pretty),MinimalStyle,CSTParser.EXPR,State})
```

Comment:
don't know if this is necessary, unclear if this precompile file is doing anything useful at this point